kube/services/peregrine/peregrine-deploy.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: peregrine-deployment
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: peregrine
+    spec:
+      volumes:
+        - name: config-volume
+          secret:
+            secretName: "peregrine-secret"
+        - name: cert-volume
+          secret:
+            secretName: "cert-peregrine-service"
+        - name: ca-volume
+          secret:
+            secretName: "service-ca"
+      containers:
+        - name: peregrine
+          image: quay.io/cdis/peregrine:master
+          ports:
+          - containerPort: 80
+          - containerPort: 443
+          env:
+          - name: DICTIONARY_URL
+            valueFrom:
+              configMapKeyRef:
+                name: global
+                key: dictionary_url
```

Comment:
Can we set this to the datadictionary url so the service will work when it first comes up?

Reply:
Just kidding.